mojaloop-technical-overview/event-framework/README.md

```diff
@@ -47,8 +47,8 @@ The purpose of the Event Framework is to provide a standard unified architecture
     "metadata": {
         "event": {
             "id": "3920382d-f78c-4023-adf9-0d7a4a2a3a2f",
-            "type": "position",
-            "action": "commit",
+            "type": "log",
+            "action": "start",
```

Comment:
Note that this is not a valid action for an event type of `log`. See the data model specification below. 

Refer to https://github.com/mdebarros/documentation/tree/feature/%23767DesignEvent%2CAuditingandErrorFrameworkProposal/mojaloop-technical-overview/event-framework#3215-enum-eneventtype for details

Reply:
Ok, got it.

Reply:
Changed to trace/start